src/main/java/io/scif/services/DefaultFormatService.java

```diff
@@ -277,8 +280,12 @@ public Writer getWriterForLocation(final Location fileId)
 		}
 
 		if (w == null) {
-			throw new FormatException(
-				"No compatible output format found for extension: " + fileId);
+		    Set<String> suffixes = new TreeSet<>();
+		    for (Format f : getOutputFormats()) {
+		        suffixes.addAll(Arrays.asList(f.getSuffixes()));
+		    }
+		    throw new FormatException("No compatible output format found for extension: " + fileId + "\n"  +
+		     "Compatible output formats: " + suffixes);
```

Comment:
I would say "Available output formats" instead. "Compatible" here has to do with `Format` and the given extension.. the point being that there *are no* compatible formats.